ts/src/xt.ts

```diff
@@ -0,0 +1,4239 @@
+        } else if (market['inverse']) {
+            response = await this.publicInverseGetFutureMarketV1PublicQAggTicker (this.extend (request, params));
+        } else {
+            response = await this.publicSpotGetTicker24h (this.extend (request, params));
+        }
+        //
+        // spot
+        //
+        //     {
+        //         "rc": 0,
+        //         "mc": "SUCCESS",
+        //         "ma": [],
+        //         "result": [
+        //             {
+        //                 "s": "btc_usdt",
+        //                 "t": 1678172693931,
+        //                 "cv": "34.00",
+        //                 "cr": "0.0015",
+        //                 "o": "22398.05",
+        //                 "l": "22323.72",
+        //                 "h": "22600.50",
+        //                 "c": "22432.05",
+        //                 "q": "7962.256931",
+        //                 "v": "178675209.47416856"
+        //             }
+        //         ]
+        //     }
+        //
+        // swap and future
+        //
+        //     {
+        //         "returnCode": 0,
+        //         "msgInfo": "success",
+        //         "error": null,
+        //         "result": {
+        //             "t": 1678172848572,
+        //             "s": "btc_usdt",
+        //             "c": "22415.5",
+        //             "h": "22590.0",
+        //             "l": "22310.0",
+        //             "a": "623654031",
+        //             "v": "1399166074.31675",
+        //             "o": "22381.5",
+        //             "r": "0.0015",
+        //             "i": "22424.5",
+        //             "m": "22416.5",
+        //             "bp": "22415",
+        //             "ap": "22415.5"
+        //         }
+        //     }
+        //
+        const ticker = this.safeValue (response, 'result');
+        if (market['spot']) {
+            return this.parseTicker (ticker[0], market);
+        }
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols: string[] = undefined, params = {}) {
+        /**
+         * @method
+         * @name xt#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @see https://doc.xt.com/#market10ticker24h
+         * @see https://doc.xt.com/#futures_quotesgetAggTickers
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the xt api endpoint
+         * @returns {object} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        let market = undefined;
```

Comment:
something is off here, can you check?
```
 p xt fetchTickers '["BTC/USDT"]'          
Python v3.10.9
CCXT v3.0.50
xt.fetchTickers(['BTC/USDT'])
{}
```

Reply:
This and fetchBidsAsks are working with one symbol defined for javascript and typescript, are you calling pip examples/py/cli with these? I'm just curious what the actual command you're using is because I don't usually test python directly through the terminal like that

Reply:
I'm using the python cli typically (`python cli.py args`)

Reply:
@Dan-krm this is what I get in python 
```
p xt fetchBidsAsks  '["BTC/USDT"]' --verbose 
Python v3.10.9
CCXT v3.0.51
xt.fetchBidsAsks(['BTC/USDT'])

fetch Request: xt GET https://sapi.xt.com/v4/public/ticker/book?symbols=%5B%27BTC%2FUSDT%27%5D RequestHeaders: {'Content-Type': 'application/json', 'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate'} RequestBody: None

fetch Response: xt GET https://sapi.xt.com/v4/public/ticker/book?symbols=%5B%27BTC%2FUSDT%27%5D 200 ResponseHeaders: {'Date': 'Wed, 05 Apr 2023 17:13:14 GMT', 'Content-Type': 'application/json', 'Content-Length': '43', 'Connection': 'keep-alive', 'X-RateLimit-Remaining': '19', 'X-RateLimit-Requested-Tokens': '1', 'X-RateLimit-Burst-Capacity': '20', 'X-RateLimit-Replenish-Rate': '10', 'CF-Cache-Status': 'DYNAMIC', 'Server': 'cloudflare', 'CF-RAY': '7b336ba95dff772c-LHR'} ResponseBody: {"rc":0,"mc":"SUCCESS","ma":[],"result":[]}
{}
```

Reply:
It looks like the `'symbols'` parameter isn't working properly (verbose output in JS with the symbols still there returned all symbols instead of just the one defined), I removed it and it looks like the methods are working properly now:
```
python3 examples/py/cli.py xt fetchBidsAsks '["BTC/USDT"]'
Python v3.8.10
CCXT v3.0.51
xt.fetchBidsAsks(['BTC/USDT'])
{'BTC/USDT': {'ask': 28037.06,
              'askVolume': 2.212096,
              'average': None,
              'baseVolume': None,
              'bid': 28037.04,
              'bidVolume': 0.933188,
              'change': None,
              'close': None,
              'datetime': '2023-04-06T07:19:25.987Z',
              'high': None,
              'info': {'ap': '28037.06',
                       'aq': '2.212096',
                       'bp': '28037.04',
                       'bq': '0.933188',
                       's': 'btc_usdt',
                       't': '1680765565987'},
              'last': None,
              'low': None,
              'open': None,
              'percentage': None,
              'previousClose': None,
              'quoteVolume': None,
              'symbol': 'BTC/USDT',
              'timestamp': 1680765565987,
              'vwap': None}}
```